IBPSA/Media/Steam.mo

```diff
@@ -0,0 +1,855 @@
+    "Return specific heat capacity at constant volume"
+  algorithm
+    cv := cv_pT(
+          state.p,
+          state.T);
+    annotation (Inline=true);
+  end specificHeatCapacityCv;
+
+  redeclare function extends density_derh_p
+    "Density derivative by specific enthalpy"
+  algorithm
+    ddhp := Modelica.Media.Water.IF97_Utilities.ddhp(
+          state.p,
+          state.h,
+          phase,
+          region);
+    annotation (Inline=true);
+  end density_derh_p;
+
+  redeclare function extends density_derp_h
+    "Density derivative by pressure"
+  algorithm
+    ddph := Modelica.Media.Water.IF97_Utilities.ddph(
+          state.p,
+          state.h,
+          phase,
+          region);
+    annotation (Inline=true);
+  end density_derp_h;
+
+  redeclare replaceable function extends isentropicExponent
+    "Return isentropic exponent"
+  algorithm
+    gamma := Modelica.Media.Water.IF97_Utilities.isentropicExponent_pT(
+          state.p,
+          state.T,
+          region);
+    annotation (Inline=true);
+  end isentropicExponent;
+
+  redeclare replaceable function extends isothermalCompressibility
+    "Isothermal compressibility of water"
+  algorithm
+    kappa := Modelica.Media.Water.IF97_Utilities.kappa_pT(
+          state.p,
+          state.T,
+          region);
+    annotation (Inline=true);
+  end isothermalCompressibility;
+
+  redeclare replaceable function extends isobaricExpansionCoefficient
+    "Isobaric expansion coefficient of water"
+  algorithm
+    beta := Modelica.Media.Water.IF97_Utilities.beta_pT(
+          state.p,
+          state.T,
+          region);
+  end isobaricExpansionCoefficient;
+
+  redeclare replaceable function extends isentropicEnthalpy
+    "Compute h(s,p)"
+  algorithm
+    h_is := Modelica.Media.Water.IF97_Utilities.isentropicEnthalpy(
+          p_downstream,
+          specificEntropy(refState),
+          0);
+    annotation (Inline=true);
+  end isentropicEnthalpy;
+
+  redeclare replaceable function extends molarMass
+    "Return the molar mass of the medium"
+  algorithm
+    MM := fluidConstants[1].molarMass;
+  end molarMass;
+  // Saturation state functions
+
+  redeclare replaceable function extends saturationTemperature_p
+    "Return saturation temperature"
+  algorithm
+    T := Modelica.Media.Water.IF97_Utilities.BaseIF97.Basic.tsat(p);
+    annotation (Inline=true);
+  end saturationTemperature_p;
+
+  redeclare replaceable function extends enthalpyOfSaturatedLiquid_sat
+    "Return enthalpy of saturated liquid"
+  algorithm
+    hl := Modelica.Media.Water.IF97_Utilities.BaseIF97.Regions.hl_p(sat.psat);
+    annotation (Inline=true);
+  end enthalpyOfSaturatedLiquid_sat;
+
+  redeclare replaceable function extends enthalpyOfSaturatedVapor_sat
+    "Return enthalpy of saturated vapor"
+  algorithm
+    hv := Modelica.Media.Water.IF97_Utilities.BaseIF97.Regions.hv_p(sat.psat);
+    annotation (Inline=true);
+  end enthalpyOfSaturatedVapor_sat;
```

Comment:
See above comment about whether the functions ending in `_sat` are really needed.